modules/test/playwright/tests/account-admin-web/accounts.spec.ts

```diff
@@ -191,3 +191,34 @@ test('LPD-18484 Add account website', async ({
 		page.getByRole('cell', {name: 'https://www.website.com'})
 	).toBeVisible();
 });
+
+test('LPD-25948 Can search for account group', async ({
```

Comment:
please create a new file (inside the same folder) named accountGroups.spec.ts with this test